CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Change Log for MIGraphX
+
+Full documentation for MIGraphX is available at [MIGraphX Github Pages](https://rocmsoftwareplatform.github.io/AMDMIGraphX/doc/html/).
+
+## MIGraphX 2.5 for ROCm 5.5.0
+
+### Added
```

Comment:
```suggestion
### Additions
```

Reply:
I won't change this keyword as it is part of the ROCm Changelog architecture